### PR TITLE
Azure FrontDoor v2 detectors module

### PR DESCRIPTION
**Pull request by maxpoullain**
- Add Azure FrontDoor Premium/Standard detectors via [FAME](https://github.com/claranet/fame) metrics.

### CODE REVIEW

**Comment by Shr3ps** [comment]
Hum, the directory name cannot start with "integration" since it's not based on the Azure integration 🤔 
`claranet`? `fame`? `custom`?

File: modules/integration_azure-frontdoor-v2/conf/readme.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  The detectors of this module uses metrics from the [FAME](https://github.com/claranet/fame) tool for Azure.  
+  Check its documentation to install and configure it appropriately.
+
+  The Azure Front Door Premium/Standard resource is not yet integrated with the Azure SFX integration.  
+  You will find the FAME queries used by this module below.
+
+  ### Pre-requisites FAME queries
+
+  Here are the must have FAME queries for your Front Door v2 resources :
```
Comment (Shr3ps) [inline modules/integration_azure-frontdoor-v2/conf/readme.yaml:22]:
```suggestion
  Here are the must have FAME queries for your Front Door v2 resources:
```

**Review by Shr3ps — DISMISSED** [review]

**Comment by xp-1000** [comment]
@Shr3ps @BzSpi can you review this?

**Review by Shr3ps — APPROVED** [review]

File: modules/fame_azure-frontdoor-v2/conf/02-probes_errors.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+transformation: true
+aggregation: true
+
+signals:
+  A:
+    metric: fame.azure.frontdoor.probe_response_status
+    extrapolation: 'zero'
+    rollup: 'sum'
+    filter: "filter('http_status_code', '5**') or filter('http_status_code', '404') or filter('result', 'OriginTimeout') or filter('result', 'DNSNameNotResolved')"
```
Comment (BzSpi) [inline modules/fame_azure-frontdoor-v2/conf/02-probes_errors.yaml:12]: Can't we heck if status != 200 instead ?

File: modules/fame_azure-frontdoor-v2/conf/03-cache_hit_rate.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+signals:
+  A:
+    metric: fame.azure.frontdoor.cache_hit_rate
+    extrapolation: 'last_value'
+  signal:
+    formula: A.fill(0).mean(by="endpoint")
+rules:
+  warning:
+    threshold: 60
```
Comment (BzSpi) [inline modules/fame_azure-frontdoor-v2/conf/03-cache_hit_rate.yaml:15]: I'm not sure having a cache hit rate < 60% should be a global alerting rule.

**Review by BzSpi — CHANGES_REQUESTED** [review]

**Comment by BzSpi** [comment]
We will fallback to Azure metrics

**Pull request closed without merging**